odoo/addons/base/static/src/scss/onboarding.scss

```diff
@@ -1,409 +1,164 @@
-$o-onboarding-dot-size: 20px;
-$o-onboarding-dot-timing: 0.5s;
+// = Onboarding Panel
+// ============================================================================
+.o_onboarding_main {
+    $o-onboarding-base-time: 0.5s;
+    $o-onboarding-dot-size: 20px;
+    $o-onboarding-step-width: map-get($container-max-widths, 'lg') / 4;
+
+    $o-onboarding-colors: (
+        'blue': (
+            'bg': (#60ABC3, #4187AE),
+            'ui': (#fcb157, #FBEEC8),
+        ),
+        'orange': (
+            'bg': (#DA9D77, #B9684F),
+            'ui': (#0774f4, #68B3C9),
+        ),
+        'violet': (
+            'bg': (#6070C3, #475DA4),
+            'ui': (#F39D9B, #FBEEC8),
+        ),
+    );
```

Comment:
Does those variables definition be wrapped inside `.o_onboarding_main`?

Also, shouldn't they be extracted in a `*.variables.scss` file?

Reply:
> Does those variables definition be wrapped inside `.o_onboarding_main`?

Yes. These are local variables, needed for SASS "within-component" computations only.

> Also, shouldn't they be extracted in a `*.variables.scss` file?

Nope and for the same reason as before. We should expose variable when:
- The component needs to be customised in Enterprise
- Variables are used by other components (eg. `burgerMenu` uses variables exposed by `navbar`)

... at least this is what has been decided with FrameworkJS, but I don't think that something changed on this side meanwhile.